src/bodies.cpp

```diff
@@ -595,34 +595,40 @@ void bodies::Box::computeBoundingCylinder(BoundingCylinder& cylinder) const
   cylinder.radius = sqrt(a * a + b * b);
 }
 
+// for some reason, the O2 optimization screws up this function which then gives wrong results
```

Comment:
What?

Reply:
No idea... That's what I observed. Turning off O2 optimization solved the issue.